lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/participants/codeaction/MavenNoGrammarConstraintsCodeAction.java

```diff
@@ -22,7 +22,7 @@
 import org.eclipse.lsp4j.jsonrpc.CancelChecker;
 
 public class MavenNoGrammarConstraintsCodeAction implements ICodeActionParticipant {
-	public static final String XSI_VALUE = " xmlns=\"http://maven.apache.org/POM/4.0.0\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" //$NON-NLS-1$
+	public static final String XSI_VALUE = " xmlns=\"http://maven.apache.org/POM/4.0.0\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"" + System.lineSeparator() //$NON-NLS-1$
```

Comment:
Here we sould read the current file and apply the same line separator as existing one on that line. if no line is present, take the default one.

Reply:
OK